libs/output-mapping/src/Configuration/Table/BaseConfiguration.php

```diff
@@ -47,18 +50,114 @@ public static function configureNode(ArrayNodeDefinition $node): void
-                    ->defaultValue('eq')
+                    ->defaultValue(self::DEFAULT_DELETE_WHERE_OPERATOR)
                     ->beforeNormalization()
                         ->ifInArray(['', null])
                         ->then(function () {
-                            return 'eq';
+                            return self::DEFAULT_DELETE_WHERE_OPERATOR;
                         })
                     ->end()
                     ->validate()
-                    ->ifNotInArray(['eq', 'ne'])
+                    ->ifNotInArray(self::ALLOWED_DELETE_WHERE_OPERATORS)
                         ->thenInvalid('Invalid operator in delete_where_operator %s.')
                     ->end()
                 ->end()
+                ->arrayNode('delete_where')
+                    ->prototype('array')
+                        ->children()
+                            ->scalarNode('changed_since')->end()
+                            ->scalarNode('changed_until')->end()
+                            ->arrayNode('where_filters')
+                                ->prototype('array')
+                                    ->children()
+                                        ->scalarNode('column')->isRequired()->cannotBeEmpty()->end()
+                                        ->scalarNode('operator')
+                                            ->defaultValue(self::DEFAULT_DELETE_WHERE_OPERATOR)
+                                            ->validate()
+                                                ->ifNotInArray(self::ALLOWED_DELETE_WHERE_OPERATORS)
+                                                ->thenInvalid(
+                                                    'Invalid operator %s. Valid values are "' .
+                                                        implode('" or "', self::ALLOWED_DELETE_WHERE_OPERATORS) . '".',
+                                                )
+                                            ->end()
+                                        ->end()
+                                        ->arrayNode('values_from_set')
+                                            ->prototype('scalar')->end()
+                                        ->end()
+                                        ->arrayNode('values_from_workspace')
+                                            ->children()
+                                                ->scalarNode('id')->isRequired()->cannotBeEmpty()->end()
+                                                ->scalarNode('table')->isRequired()->cannotBeEmpty()->end()
+                                                ->scalarNode('column')->end()
+                                            ->end()
+                                        ->end()
+                                        ->arrayNode('values_from_storage')
+                                            ->children()
+                                                ->scalarNode('bucket')->isRequired()->cannotBeEmpty()->end()
```

Comment:
Porad premyslim, jestil by to nemelo byt
- workspace_id
- bucket_id 

🤔

Reply:
no minimálně u toho workspacu bych nenechával jen ID - to může být zavádějící a bucket_id/workspace_id je pravda nejlepší

Reply:
@ondrajodas Nechapu ten komentar :) Rikas, nechal bych u workspace jen ID. A pak pises, ze ten novy navrh je nejlepsi :D

Reply:
no já právě psal nenechal :)